MeshLib/CoordinateSystem.h

```diff
@@ -47,8 +47,7 @@ class CoordinateSystem
     explicit CoordinateSystem(const Element &ele);
 
     /// Decides a coordinate system from a bounding box
-    template <class T>
-    explicit CoordinateSystem(const GeoLib::AABB<T> &bbox) : _type(getCoordinateSystem(bbox)) {}
+    explicit CoordinateSystem(const GeoLib::AABB &bbox) : _type(getCoordinateSystem(bbox)) {}
```

Comment:
Yes, the `CoordinateSystem` describes the dimensionality of a point set but also if a 2D set lies in a plane parallel to the xz plane (or any other plane), or 1D set is parallel to any of the x, y, z axes.